splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -1383,6 +1383,8 @@ public static void SYSCS_SAVE_SOURCECODE(String schemaName, String objectName, S
     }
 
     public static void SET_PURGE_DELETED_ROWS (String schemaName, String tableName, String enable) throws Exception{
+        schemaName = schemaName.toUpperCase();
```

Comment:
Use EngineUtils.validateSchemat()/validateTable() instead, they deal with quoted names. This would also be needed in many other procedures here.

Reply:
Done. Short question though, validate{Schema, Table} do not seem to be idempotent. Are they?

Reply:
No, they are not.